packages/generator/test/test-util/create-generator-options.ts

```diff
@@ -3,7 +3,7 @@ import {
   cliOptions,
   ParsedGeneratorOptions
 } from '../../src/options';
-import { parseOptions } from '../../src/options/options-parser';
+import { parseOptions } from '../../../generator-common/src/options-parser';
```

Comment:
[q] Should this not be imported from `@sap-cloud-sdk/generator-common`?

Reply:
for things in `src` for sure. For the test it would mean that you need to compile so a new version is in `dist`. Since I execute the tests in the IDE directly in many cases I would like to avoid this.

Reply:
Understood. Turborepo does take care of this (and probably compiles this anyways if you run your tests through `yarn test ...` instead of `jest ...`), but I understand the reason.